.github/workflows/test-e2e.yml

```diff
@@ -32,20 +30,24 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-node-
       - name: Install npm deps
-        run: npm ci
+        run: |
+          npm ci
+          npm install -g playwright-cli
+          npx playwright install --with-deps chromium
       - name: Install composer deps
         run: composer install --no-dev
       - name: Install environment
         run: |
-          npm run up:ci
-          bash ./bin/run-e2e-tests-${{matrix.env}}.sh
-      - name: Run ${{ matrix.env }} Cypress tests
+          npm run env:up
+      - name: Run the tests
+        run: |
+          npm run test:e2e:playwright
         env:
           GITHUB_TOKEN: ${{ secrets.BOT_TOKEN }}
-        uses: cypress-io/github-action@v2
+      - name: Archive test results
+        if: failure()
+        uses: actions/upload-artifact@v4
         with:
-          env: host=localhost,port=8080
-          browser: chrome
-          install: ${{ ! steps.npm-and-build-cache.outputs.cache-hit }}
-          headless: true
-          spec: cypress/integration/${{ matrix.env }}/**/*
\ No newline at end of file
+          name: e2e-playwright-results
+          path: artifacts
+          retention-days: 3
```

Comment:
I don't think this folder exists during the build and this might fail. We can have a folder inside the testing folder to store this. We can ignore all stored files from that folder except for a `.gitkeep` file.
Also I think having the retention-days changed to 1 should be enough, there is no reason to store the logs for more than a day, as the issues should be adressed as soon as possible and you can always re-run the job.

Another thing to keep in mind is to make sure we ignore the e2e testing folders and reports from the `.distignore` file.